routes/w3c/group.js

```diff
@@ -23,7 +23,7 @@ const API_KEY = env("W3C_API_KEY");
  * @property {"PP2017" | "PP2020" | null} [Group.patentPolicy]
  */
 /** @type {MemCache<Group>} */
-const cache = new MemCache(ms("2 weeks"));
+const cache = new MemCache(ms("3 days"));
```

Comment:
As per https://github.com/marcoscaceres/respec.org/issues/121#issuecomment-704083658
```suggestion
const cache = new MemCache(ms("1 day"));
```